chain/jsonrpc/src/lib.rs

```diff
@@ -67,6 +66,8 @@ impl Default for RpcLimitsConfig {
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct RpcConfig {
     pub addr: String,
+    // If provided, will start an http server exporting only Prometheus metrics on that address.
+    pub monitoring_addr: Option<String>,
```

Comment:
Can we change the variable and the flag to be `prometheus` rather than `monitoring`